src/warnet/bitcoin.py

```diff
@@ -187,7 +187,6 @@ def get_messages(tank_a: str, tank_b: str, chain: str):
                 file_path = f"{base_dir}/{dir_name}/{file}"
                 # Fetch the file contents from the container
                 cmd = f"kubectl exec {tank_a} -- cat {file_path}"
-                import subprocess
```

Comment:
Wasn't there a reason this was in here?

Reply:
its up top now, i thought that might be what the linter was complaining about since it does not inform me, the guy trying to pass CI, what it was complaining about 🔥